tests/test_compress.py

```diff
@@ -0,0 +1,16 @@
+from group_12.group_12 import shrink, sharpen, compress
```

Comment:
You should import from the file you created, and only with the functions in that file

Reply:
I see, thanks!